helpers.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  // Return a stylised embed for consistancy - this allows you to add additional properties like fields
   sendEmbedMessage: function (Discord, logger, message, title, description, colour = '#F4B400') {
```

Comment:
This method no longer sends an embed message. Instead it generates an embed, ready for sending. I suggest this be renamed to something like generateEmbedMessage so that it more accurately reflects the new functionality.

Reply:
You're absolutely right, I'll get that changed.